api/proto/external.proto

```diff
@@ -53,31 +53,54 @@ message Ed25519Signature {
     bytes data = 1;
 }
 
-// Complete AccountKey, containing the pair of secret keys, which can be used
-// for spending, and optionally some Fog related info that can be used for spending.
-//
-// This matches the Rust `transaction::AccountKey` struct.
+/// Complete AccountKey, containing the pair of secret keys, which can be used
+/// for spending, and optionally some Fog related info that is used to form
+/// public addresses for accounts that sign up with Fog service.
+///
+/// This matches the Rust `transaction::AccountKey` struct.
 message AccountKey {
-    // Private key 'a' used for view-key matching.
+    /// Private key 'a' used for view-key matching.
     RistrettoPrivate view_private_key = 1;
 
-    // Private key `b` used for spending.
+    /// Private key `b` used for spending.
     RistrettoPrivate spend_private_key = 2;
 
-    // Optional url of fog server. Empty string when not in use.
+    /// Optional url of fog report server.
+    /// Empty string when not in use, i.e. for accounts that don't have fog service.
```

Comment:
If this is the report server, can we call it `fog_report_url`?

Reply:
yah i'm on board with that